prometheuscollector/prometheuscollector.go

```diff
@@ -46,11 +46,11 @@ var (
 )
 
 type Collector struct {
-	metrics tusd.Metrics
+	metrics *tusd.Metrics
```

Comment:
But the pointer here remains needed, right?

![2017-02-27-221005_2560x1080_scrot](https://cloud.githubusercontent.com/assets/3864879/23380100/d1f946b0-fd39-11e6-95f6-6f9ce2b72a74.png)